### PR TITLE
Fix tests with pygments 2.12

### LINKED ISSUE #28: Tests are failing with Pygments 2.12.0
In Fedora, we've discovered that two tests of stack_data are not compatible with the latest Pygments. I did a bisection of the commits between Pygments' releases and the problem is caused by this change: https://github.com/pygments/pygments/commit/1458c6669bbc9121eb80b08a7112d81899da7344 

I'll try to open a PR soon.

### PR DESCRIPTION
**Pull request by frenzymadness**
Fixes #28

### CODE REVIEW

**Comment by frenzymadness** [comment]
I guess that the Coveralls failures are expected even the coverage is still the same.

File: tests/test_core.py [hunk 1]
```diff
@@ -552,20 +553,20 @@ def test_absolute_filename():
     assert frame_info.filename == full_filename
 
 
-def test_executing_style_defs():
+@pytest.mark.parametrize("expected",
```
Comment (alexmojaki) [inline tests/test_core.py:556]: parametrize seems like the wrong approach here. This isn't a family of related tests, it's a bunch of assertions about a single string.

File: tests/test_core.py [hunk 2]
```diff
@@ -552,20 +553,20 @@ def test_absolute_filename():
...
+    [
+        r".c { color: #(999999|ababab); font-style: italic }",
+        r".err { color: #a61717; background-color: #e3d2d2 }",
+        r".c-ExecutingNode { color: #(999999|ababab); font-style: italic; background-color: #ffff00 }",
+        r".err-ExecutingNode { color: #a61717; background-color: #ffff00 }",
+    ]
+)
+def test_executing_style_defs(expected):
     style = style_with_executing_node("native", "bg:#ffff00")
```
Comment (alexmojaki) [inline tests/test_core.py:565]: Maybe a better solution would be to switch from `native` to a different style that hasn't changed recently.

Comment (frenzymadness) [reply tests/test_core.py:565]: Might be, I don't know Pygments well enough. Feel free to suggest a change.

**Pull request merged**